### PR TITLE
home work 2

### PR DESCRIPTION
**Pull request by dima666Sik**
Home work 2 zd 1-4

### CODE REVIEW

File: home_work_2_zd_2.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
+let number = -4;
+
+function sumNumb(number) {
```
Comment (StanislawVictorovich) [inline home_work_2_zd_2.js:3]: function is not working properly

File: home_work_2_zd_3.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+let str = "JavaScript";
+
+function reverse(str) {
```
Comment (StanislawVictorovich) [inline home_work_2_zd_3.js:3]: function is not working properly. For example if we execute function with the string that contents numbers.

File: home_work_2_zd_4.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+let newStr = "";
+function concat(str) {
```
Comment (StanislawVictorovich) [inline home_work_2_zd_4.js]: this is not a closure realization. Need rewrite. Don't use global scope

File: home_work_2_zd_2.js (hunk 1 above)
Comment (dima666Sik) [reply home_work_2_zd_2.js:3]: Done

File: home_work_2_zd_3.js (hunk 2 above)
Comment (dima666Sik) [reply home_work_2_zd_3.js:3]: Done

File: home_work_2_zd_4.js (hunk 3 above)
Comment (dima666Sik) [reply home_work_2_zd_4.js]: Done

**Pull request merged**